config/bool.go

```diff
@@ -0,0 +1,97 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+)
+
+// GetBoolT returns the value associated with the given key as a boolean.
```

Comment:
After reading this doc block I was still unsure what the `T` was in the name or why it was returning `true` by default when the env var did not exist. I wonder if this should mention something about falling back to the default of true? If this is a common thing in Go libraries then feel free to ignore my this comment.

Reply:
Nah, I think you're right - these docs could be improved.